R/03-geojob-obj.R

```diff
@@ -110,7 +110,7 @@ setMethod(f = "xml<-",signature = "geojob", definition = function(.Object, value
 #'@rdname geojob-methods
 #'@aliases xml,geojob-method
 #'@examples
-#'xml <- "<foo> <bar> text <baz/> </bar> </foo>"
+#'xml <- "<foo version=\"1.0.0\"> <bar> text <baz/> </bar> </foo>"
```

Comment:
The algorithm version has to come out of the xml.  I think with any GDP xml it would be there?  I could check that it exists first though and leave it blank if not.